test/modules/packages/blas/test_blas1.chpl

```diff
@@ -0,0 +1,646 @@
+use Random;
```

Comment:
I think you should consider adding some tests (not necessarily in this file) that **should** fail (e.g., mismatching array sizes).